sparse/src/KokkosSparse_par_ilut.hpp

```diff
@@ -330,8 +330,13 @@ void par_ilut_numeric(KernelHandle* handle, ARowMapType& A_rowmap, AEntriesType&
     KokkosKernels::Impl::throw_runtime_exception(os.str());
   }
 
-  KK_REQUIRE_MSG(KokkosSparse::Impl::isCrsGraphSorted(L_rowmap, L_entries), "L is not sorted");
-  KK_REQUIRE_MSG(KokkosSparse::Impl::isCrsGraphSorted(U_rowmap, U_entries), "U is not sorted");
+  // These are outputs, so we don't want whatever junk that may be in these to
+  // cause the isSorted check to fail.
+  Kokkos::deep_copy(L_entries, 0);
+  Kokkos::deep_copy(U_entries, 0);
+
+  KK_REQUIRE_MSG(KokkosSparse::Impl::isCrsGraphSorted(L_rowmap, L_entries), "L rowmap is not sorted");
```

Comment:
If L_entries and U_entries are filled with 0, and their rowmaps are still valid, then isCrsGraphSorted will always return true. But if the rowmap contains garbage values then it could segfault.

To test if L_rowmap itself is sorted, there's ``Kokkos::Experimental::is_sorted(exec_space(), L_rowmap)``.